router.go

```diff
@@ -186,17 +186,30 @@ func (res CSVResponse) Write(w http.ResponseWriter, r *http.Request) {
 	w.Write(res.CSV)
 }
 
+type LogType string
+
+const (
+	LogTypeInfo  LogType = "INFO"
+	LogTypeWarn  LogType = "WARN"
+	LogTypeError LogType = "ERROR"
+)
+
 // APIResponse response data representation for API
 type APIResponse struct {
 	Error  string      `json:"error,omitempty"`
 	Status string      `json:"status,omitempty"`
 	Data   interface{} `json:"data,omitempty"`
+	Type   LogType     `json:"type,omitempty"`
 }
 
 // Write - Reponse interface implementation
 func (res APIResponse) Write(w http.ResponseWriter, r *http.Request) {
-	if res.Status == "ERROR" {
+	if res.Type == LogTypeError {
 		logger.Errorf("[API][PATH: %s]:: Error handling request. ERROR: %s. User agent: %s", r.RequestURI, res.Error, r.Header.Get("User-Agent"))
+	} else if res.Type == LogTypeWarn {
+		logger.Warnf("[API][PATH: %s]:: Warning handling request. WARN: %s. User agent: %s", r.RequestURI, res.Error, r.Header.Get("User-Agent"))
+	} else {
+		logger.Warnf("[API][PATH: %s]:: Info handling request. INFO: %s. User agent: %s", r.RequestURI, res.Error, r.Header.Get("User-Agent"))
```

Comment:
are we logging Info as logger.Warnf here??

Reply:
@DibeshMSShrestha made the fix